src/orders/models/order.py

```diff
@@ -152,3 +153,17 @@ def unship(self) -> None:
         from orders.services import OrderUnshipper
 
         OrderUnshipper(self)()
+
+    def get_payment_method(self) -> str:
```

Comment:
```suggestion
    def get_readable_payment_method_name(self) -> str:
```

Этот метод возвращает название метода оплаты, а не метод оплаты. К тому же у нас вообще нет метода оплаты, и чтобы не запутывать будущего программиста, лучше бы сразу дать понять из названия метода, что мы это делаем только для человека, а не для робота.